Twoliter.lock

```diff
@@ -1,17 +1,17 @@
 schema-version = 1
 release-version = "1.21.0"
-digest = "y1FRkXnpl5De27FxP96g8yuAZojLSff31XG6PKFUuZY="
+digest = "xrDJu9xcRcdcb2lrWOH9P9gXHfe83eO68IRJRFxlZsg="
 
 [sdk]
 name = "bottlerocket-sdk"
-version = "0.42.0"
+version = "0.43.0"
 vendor = "bottlerocket"
-source = "public.ecr.aws/bottlerocket/bottlerocket-sdk:v0.42.0"
-digest = "myHHKE41h9qfeyR6V6HB0BfiLPwj3QEFLUFy4TXcR10="
+source = "public.ecr.aws/bottlerocket/bottlerocket-sdk:v0.43.0"
+digest = "1zKyWC/pXdpzKfNj1Xp1YvEWdyltI7EhAeiNwlikBz8="
```

Comment:
Should updates to the SDK `source` and  `digest` in the lock file be included in the commit bumping the SDK?

Reply:
I think so. The build will fail if you don't update the `.lock` file after updating the `Twoliter.toml`.

Reply:
Sorry let me rephrase: Currently, the SDK `source` and `digest` updates in the lock file are included in the commit bumping the core-kit version. Should this be moved to the commit bumping the SDK version instead?

Reply:
I see what you mean. Yeah I'm not sure. I think there is some dependency between the core-kit and sdk version which requires both, the main repo and the kit to be on the same sdk version so taking just one of these changes would lead to an undefined state as well.